apps/healthlocker/web/controllers/oxleas_adhd/login_controller.ex

```diff
@@ -20,7 +20,7 @@ defmodule Healthlocker.OxleasAdhd.LoginController do
           "clinician" ->
             conn
             |> put_flash(:info, "Logged in as clinician")
-            |> redirect(to: page_path(conn, :index))
+            |> redirect(to: page_path(conn, :caseload))
```

Comment:
@katyedwards This was a good attempt but you needed to update the path that we redirect to, not the route in that path. 

It should look like this...
```
|> redirect(to: caseload_path(conn, :index))
```

I'll update it now but just wanted to point it out to you 👍